docs/config.md

```diff
@@ -275,6 +276,14 @@ an implicit default when `null` or missing from the config file:
 - `null`/missing (apply the default value defined in go-ipfs sources)
 - an integer between `-2^63` and `2^63-1` (i.e. `-9223372036854775808` to `9223372036854775807`)
 
+### `optionalBytes`
+
+Optional Bytes allow specifying some number of bytes which has
+an implicit default when `null` or missing from the config file:
+
+- `null`/missing (apply the default value defined in go-ipfs sources)
+- a string value indicating the number of bytes (e.g. 1B, 10kB, 20 MiB, 2GB, ...)
```

Comment:
I never remember which is which, and even if I do, I don't trust apps and [check](https://github.com/dustin/go-humanize/blob/afde56e7acacd811f6c94228c2c61af2b0e93158/bytes.go#L88-L110) if they correctly support one or another. :see_no_evil: 
By writing it down, we save everyone a lot of time on double-checking:
```suggestion
- a string value indicating the number of bytes, including human readable representations:
  - [SI sizes](https://en.wikipedia.org/wiki/Metric_prefix#List_of_SI_prefixes) (metric units, powers of 1000), e.g. `1B`, `2kB`, `3MB`, `4GB`, `5TB`, …)
  - [IEC sizes](https://en.wikipedia.org/wiki/Binary_prefix#IEC_prefixes) (binary units, powers of 1024), e.g. `1B`, `2KiB`, `3MiB`, `4GiB`, `5TiB`, …)
```